types/draft-js-custom-styles/tslint.json

```diff
@@ -0,0 +1,5 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
The `rules` property is unnecessary, please remove it.

Reply:
Done